src/angular-business/chip/chip-input/chip-input.component.ts

```diff
@@ -254,7 +288,9 @@ export class ChipInputComponent extends SbbChipsMixinBase
   }
 
   /**
-   * Selects a given value if the action doesn't refer to an autocomplete option
+   * Selects a given value if the action doesn't refer to an autocomplete option.
+   * TODO: Prefix with _
```

Comment:
Wouldn't it be better to already prefix it with "_" and provide a getter and setter to onEnter?